### PR TITLE
[DE-248] DE-432 Extract Bank Account and Payment Profiles

### PR DESCRIPTION
**Pull request by michalpierog**
[DE-248] DE-432 Extract Bank Account and Payment Profiles

[DE-248]: https://maxioevolution.atlassian.net/browse/DE-248?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: components/schemas/Read-Payment-Profile-Response.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title: Read Payment Profile Response
+type: object
+properties:
+  payment_profile:
+    oneOf:
```
Comment (maciej-nedza) [inline components/schemas/Read-Payment-Profile-Response.yaml:5]: before it was anyOf

Comment (michalpierog) [reply components/schemas/Read-Payment-Profile-Response.yaml:5]: I know but it seems it was invalid. 
I think that Payment-Profile can contain either Bank Account or Credit Card properties but not both.

File: components/schemas/List-Payment-Profiles-Response.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+title: List Payment Profiles Response
```
Comment (maciej-nedza) [inline components/schemas/List-Payment-Profiles-Response.yaml:1]: Maybe list payment profiles item or something, as it's item in response array?

Comment (maciej-nedza) [inline components/schemas/List-Payment-Profiles-Response.yaml:1]: I'd name it "List Payment Profiles Response Item" and the payment_profile model inside "List Payment Profile Item", as the current name to me hints it as list

**Review by maciej-nedza — APPROVED** [review]
LGTM,
I feel like some of these responses are lacking fields, for sure List-Payment-Profiles is, I've tested this endpoint and it returned some credit card data that's not presend in the schema, please create an issue to fix that

**Pull request merged**